extensions/amp-call-tracking/0.1/validator-amp-call-tracking.protoascii

```diff
@@ -0,0 +1,84 @@
+#
+# Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-call-tracking
+  html_format: AMP
+  html_format: AMP4ADS
+  tag_name: "SCRIPT"
+  spec_name: "amp-call-tracking extension .js script"
+  satisfies: "amp-call-tracking extension .js script"
```

Comment:
I believe you want to use also_requires_tag instead of satisfies/requires. Here it would be:
also_requires_tag: "amp-call-tracking"